cli/compose/schema/data/config_schema_v3.3.json

```diff
@@ -89,6 +89,7 @@
           ]
         },
         "container_name": {"type": "string"},
+        "credential_spec": {"type": "string"},
```

Comment:
It seems we could convert it: https://github.com/moby/moby/pull/32339/files#diff-9651468cafd3eee09570f3733efd7dc4R605

But the container wouldn't retain the credential_spec information, just whatever we interpreted from it.